### ISSUE TITLE
Incoming calls don't get status callbacks

### ISSUE THREAD

**Issue by jclardy** [jefflinwood/twilio_client_phonegap#28]
I noticed that incoming calls don't get status callbacks, mainly disconnect.

I fixed it by setting the delegate on an incoming connection here:
>-(void)device:(TCDevice *)device didReceiveIncomingConnection:(TCConnection *)connection {
>    self.connection = connection;
>    self.connection.delegate = self;
>    [self javascriptCallback:@"onincoming"];
>}

**Comment by jefflinwood** [comment]
Thanks! I'll see about merging that fix in.

**Comment by adnanumar** [comment]
@jclardy thanks for the point, right now i am facing this issue, can you please explain a bit further how i can modify my code in xcode. Thanks for your time.

**Comment by adnanumar** [comment]
yes i have resolved the issue while modifying "TCPlugin.m" in cordova plugin "com.phonegap.plugins.twilioclient"... i can now accept or reject incoming call correctly. thanks for the point.
